erpnext_oralcare/erpnext_oralcare/report/doctor_financial_report_query/doctor_financial_report_query.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016, Techlift and contributors
+// For license information, please see license.txt
+/* eslint-disable */
+
+frappe.query_reports["Doctor Financial report Query"] = {
```

Comment:
Remove Query from the title. Rename Old reports by adding "Old" in the title and make this report as normal title like "Doctor Financial Report" and not "Doctor Financial Report Query". Do this for all the new reports.